client/src/app/+about/about-peertube/about-peertube.component.html

```diff
@@ -0,0 +1,94 @@
+  </p>
+
+  <h4 i18n class="p2p-privacy-title">What are the consequences?</h4>
+
+  <p i18n>
+    In theory, we could imagine that someone with enough technical skills may create a script that tracks which IP is downloading which video.
+    In practice, this is much more difficult because:
+  </p>
+
+  <ul>
+    <li i18n>
+      An HTTP request has to be sent on each tracker for each video to spy.
+      If we want to spy all PeerTube's videos, we have to send as many requests as there are videos (so potentially a lot)
+    </li>
+
+    <li i18n>
+      For each request sent, the tracker returns random peers at a limited number.
+      For instance, if there are 1000 peers in the swarm and the tracker sends only 20 peers for each request, there must be at least 50 requests sent to know every peers in the swarm
+    </li>
+
+    <li i18n>
+      Those requests have to be sent regularly to know who starts/stops watching a video. It is easy to detect that kind of behaviour
+    </li>
+
+    <li i18n>
+      If an IP address is stored in the tracker, it doesn't mean that the person behind the IP (if this person exists) has watched the video
+    </li>
+
+    <li i18n>
+      The IP address is a vague information : usually, it regularly changes and can represent many persons or entities
+    </li>
+
+    <li i18n>
+      Web peers are not directly sent by the tracker: because we use WebRTC inside the web browser, the protocol is different from classic BitTorrent.
+      When you are in a web browser, you send a signal containing your IP address to the tracker that will randomly choose other peers to forward the information to.
+      See <a href="https://github.com/yciabaud/webtorrent/blob/beps/bep_webrtc.rst">this document</a> for more information
+    </li>
+  </ul>
+
+  <p i18n>
+    The worst-case scenario of an average person spying on their friends is quite unlikely.
+    There are much more effective ways to get that kind of information.
+  </p>
+
+  <h4 i18n class="p2p-privacy-title">How does PeerTube compares to YouTube?</h4>
+
+  <p i18n>
+    The privacy threat in YouTube is different from PeerTube's.
+    In YouTube's case, the platform gathers a huge amount of your personal information (not only your IP) to analyze them and track you.
+    Moreover, YouTube is owned by Google/Alphabet, a company that tracks you across many websites (via AdSense or Google Analytics).
+  </p>
+
+  <h4 i18n class="p2p-privacy-title">What can I do to limit the exposure of my IP address?</h4>
+
+  <p i18n>
+    Your IP address is public so every time you consult a website, there is a number of actors (in addition to the final website) seeing your IP in their connection logs: ISP/routers/trackers/CDN and more.
+    PeerTube is transparent about it: we warn you that if you want to keep your IP private, you must use a VPN or Tor Browser.
+    Thinking that removing P2P from PeerTube will protect your anonymity doesn't make any sense.
```

Comment:
"[…] will give you back anonymity doesn't make sense."

"any" is making your point too strong.